packages/cli-lib/errorHandlers/apiErrors.js

```diff
@@ -200,6 +201,21 @@ function logApiUploadErrorInstance(error, context) {
   logApiErrorInstance(error, context);
 }
 
+/**
+ * Logs a message for a warning resulting from filemapper API upload.
+ *
+ * @param {Object} resp
+ */
+function logApiUploadWarnings(resp, context) {
+  if (resp && resp.warnings && resp.warnings.length) {
+    const { base } = path.parse(resp.path);
```

Comment:
nit: is it cleaner to call `path.basename()` here?

Reply:
Yep, updated in 747b3ac